ui/src/test/java/edu/wpi/grip/ui/PaletteTest.java

```diff
@@ -0,0 +1,70 @@
+package edu.wpi.grip.ui;
+
+import com.google.common.eventbus.EventBus;
+import com.google.common.eventbus.Subscribe;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import com.google.inject.util.Modules;
+import edu.wpi.grip.core.AdditionOperation;
+import edu.wpi.grip.core.Operation;
+import edu.wpi.grip.core.Step;
+import edu.wpi.grip.core.events.OperationAddedEvent;
+import edu.wpi.grip.core.events.StepAddedEvent;
+import edu.wpi.grip.util.GRIPCoreTestModule;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+import org.junit.Test;
+import org.testfx.framework.junit.ApplicationTest;
+import org.testfx.util.WaitForAsyncUtils;
+
+import javax.inject.Inject;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class PaletteTest extends ApplicationTest {
+
+    @Inject private EventBus eventBus;
+
+    private Step step;
+    private final GRIPCoreTestModule testModule = new GRIPCoreTestModule();
+
+    @Override
+    public void start(Stage stage) throws Exception {
```

Comment:
I can fix it, but for the record it was copied and pasted from `MainWindowTest`.  What was the justification for throwing `Exception` there?